src/test/java/io/neonbee/test/helper/DeploymentHelper.java

```diff
@@ -72,7 +73,7 @@
             }
         }
 
-        return CompositeFuture.all(Helper.uncheckedMapper(undeployFutures)).mapEmpty();
+        return CompositeFuture.all(uncheckedMapper(undeployFutures)).mapEmpty();
     }
 
     public static boolean isVerticleDeployed(Vertx vertx, Class<? extends Verticle> verticleToCheck) {
```

Comment:
unrelated: the last filter statement can be simplified:
```
return vertx.deploymentIDs().stream()
                .flatMap(deploymentId -> ((VertxInternal) vertx).getDeployment(deploymentId).getVerticles().stream())
                .anyMatch(verticleToCheck::isInstance);
```

Reply:
Done, good catch